tests/rptest/tests/partition_state_api_test.py

```diff
@@ -129,3 +134,70 @@ def test_controller_partition_state(self):
             leader_state = leaders[0]["raft_state"]
             assert "followers" in leader_state.keys() and len(
                 leader_state["followers"]) == 4
+
+    @cluster(num_nodes=5)
+    def test_local_summary(self):
+        admin = Admin(self.redpanda)
+        n_topics = 100
+
+        def produce():
+            kafka_tools = KafkaCliTools(self.redpanda)
+            for t in self.topics:
+                kafka_tools.produce(t.name, 1, 1, acks=1)
+
+        def sumsum_eventually(**expected):
+            self.logger.debug(f"{expected=}")
+
+            # allow for system partitions other than raft0, e.g. id_allocator
+            tolerance = 10
```

Comment:
is 10 derived from default number of partitions or something? maybe make it adaptive to the current settings or describe how it should be calculated?

Reply:
done